plugins/decklink/const.h

```diff
@@ -7,22 +7,24 @@
 #define COLOR_SPACE     "color_space"
 #define COLOR_RANGE     "color_range"
 #define BUFFERING       "buffering"
+#define AUTO_START      "auto_start"
 
-#define TEXT_DEVICE                     obs_module_text("Device")
```

Comment:
I don't think this block of whitespace changes was intentional.  If it was, could you explain it?  If not, could you please revert it?